asmcomp/s390x/emit.mlp

```diff
@@ -238,10 +238,11 @@ let int_literals = ref ([] : (nativeint * int) list)
 
 (* Masks for conditional branches after comparisons *)
 
+(* bit 0 = eq, bit 1 = lt, bit 2 = gt, bit 3 = overflow*)
 let branch_for_comparison = function
-    Ceq -> 8  | Cne -> 7
-  | Cle -> 12 | Cgt -> 2
-  | Cge -> 10 | Clt -> 4
+  | Ceq -> 0b1000 | Cne -> 0b0111 (* BRNEL is 0111 rather than 0110 *)
+  | Cle -> 0b1100 | Cgt -> 0b0010
```

Comment:
Interesting observation about BRNEL.  I believe "bit 3" (overflow) is always set to 0 by integer comparison instructions, so it probably makes no difference to use `0b0111` or `0b0110`.  Still, I'll try to track down where this "7" comes from in the original code.

Reply:
Note that by "BRNEL is 0111" , I mean that the documentation I found online uses 7, not 6, for inequality -- it is not just a choice of the OCaml backend. This seems to be a standard convention.

Reply:
OK for the standard convention.